### ISSUE TITLE
Use timezones-with-oceans shapefiles

### ISSUE THREAD

**Issue by ironwallaby** [darkskyapp/tz-lookup#33]
Depends on #29 

I think we're at the point where the rebuild system can handle using the timezones-with-oceans distribution. Doing so would simplify the build script a little bit and push more of this library's logic onto @evansiroky (which is, of course, a good thing for me ;) ).

**Comment by ironwallaby** [comment]
Already done in #29.

**Comment by ironwallaby** [comment]
Turns out there's a subtlety that makes doing this difficult: we try to use less compression in coastal areas by allowing other zones to "eat into" the ocean quite a lot. Since there are legitimate land-based zones that also use the maritime zone names (e.g. `Etc/GMT+5`), the only way to know which zones are the maritime ones is to do the work of figuring it out manually.

That is, there's no benefit to this library to use the timezones-with-oceans distribution. So I'm keeping this ticket closed anyway, but it's closed because of "will not fix" rather than "done".
